src/containers/EditCardScreen.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { View, Text, Button, TextInput } from 'react-native';
+import { CheckBox } from 'react-native-elements';
+import cardService from '../services/CardService';
+
+export default class CardScreen extends React.Component {
+  static navigationOptions = {
+    title: 'Edit Card'
+  };
+  card = this.props.navigation.getParam('card');
+  state = {
+    description: this.card.description,
+    priority: this.card.priority,
+    done: this.card.done
+  };
+
+  buttonTitle = this.props.navigation.getParam('edit') ? 'Edit' : 'Add';
+
+  buttonSubmit = () => {
+    if (this.props.navigation.getParam('edit')) {
+      //service.edit
+    } else {
+      cardService.addCard(this.state).then(response => {
+        if (response.ok) {
+          this.props.navigation.pop();
+          this.props.navigation.pop();
+          this.props.navigation.navigate('Home');
```

Comment:
odrazi bez pop samo da navigira na Home ... iz drugih razloga ide to pop nije vlada najbolje razumeo ;)